pkg/providerConfigs/adjustDriver.go

```diff
@@ -32,6 +36,23 @@ func AdjustGitopsRepo(
 	apexContentExists bool,
 	useCloudflareOriginIssuer bool,
 ) error {
+	//* Compose the providers if a taskrunner.yaml file exists
+	if _, err := os.Stat(path.Join(gitopsRepoDir, taskrunnerFile)); !os.IsNotExist(err) {
+		tasks := taskrunner.Config{
+			Root:     gitopsRepoDir,
+			TaskFile: taskrunnerFile,
+			Variables: map[string]string{
+				"cloudProvider": cloudProvider,
+				"outputDir":     gitopsRepoDir,
+				"vcsProvider":   gitProvider,
+			},
+		}
+
+		if err := tasks.Exec(); err != nil {
+			return err
```

Comment:
All errors should have contextual information and be wrapped.

```suggestion
			return fmt.Errorf("unable to execute task: %w", err)
```

This prevents us if for any reason the error is non descriptive, we can still find its source. For example, what if you run the task config, reach the end of file and the error returned is `io.EOF`? The contents there are merely `EOF`.